Makefile

```diff
@@ -20,3 +22,7 @@ upload_package: format_package build_package
 # Conda env
 create_conda_env_file:
 	 conda env export > environment.yml
+
+# Reinstall genni
+reinstall_genni: build_package
+	pip uninstall genni -y && pip install dist/genni-$(GENNI_VERSION)*.whl
```

Comment:
there is no pip install command and this pip did not work for me

Reply:
Not for users.

Reply:
For this to work you would have to build the package. I use poetry for that.